src/backend/web/app.js

```diff
@@ -3,6 +3,10 @@ const express = require('express');
 const expressHandlebars = require('express-handlebars');
 const healthcheck = require('express-healthcheck');
 const cors = require('cors');
+const { ApolloServer } = require('apollo-server-express');
+
+// Required for GraphQL
+const { typeDefs, resolvers } = require('./graphql');
```

Comment:
Can we move this down to be closer to line 26, since that's where it's being used?

Reply:
Fixed